luigi/contrib/hive.py

```diff
@@ -87,6 +98,19 @@ def run_hive_script(script):
     return run_hive(['-f', script])
 
 
+def _validate_partition(partition):
+    """
+    If partition is set and it's size more than one and it's not ordered
```

Comment:
`If partition is set and its size is more than one and not ordered, then we're unable to restore its path in the warehouse`